core/storage/user/gae_models.py

```diff
@@ -1826,7 +1826,7 @@ def export_data(cls, user_id):
         return user_data
 
 
-class UserContributionScoringModel(base_models.BaseModel):
+class UserContributionProficiencyModel(base_models.BaseModel):
```

Comment:
We need to remove old instances of `UserContributionScoringModel`, can be done manualy via Google Console by @seanlip later after we are sure taht everything works correctly, just please be aware of this.

Reply:
Okay!

Reply:
Could you please file an issue for this and maybe add TODO?

Reply:
Can we remove the model of the class which doesn't exist in the codebase? (We are removing `UserContributionScoringModel` class definition through this PR so I'm not sure whether you'll be able to remove it later? If this model is not in use anywhere then ask Sean to remove it from the prod (We may need a one-off job for this, AFAIK the console doesn't allow deleting all the models of a class at once) and then merge this PR or create a new Model `UserContributionProficiencyModel` and leave `UserContributionScoringModel` as is!

Reply:
I remember that @seanlip recently removed all the `SuggestionModel` instance that were not used for a long time through the console. I think it should be possible.

Reply:
@vojtechjelinek @DubeySandeep @seanlip which old instances of the UserContributionScoringModel are we removing if the model was never a part of production code/released? Am I missing something?

Reply:
Please note that **there are no such models in production**, so no extra steps need to be done and no issue needs to be filed.